packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -227,3 +230,71 @@ const OrganizationListPageList = (props: { onCreateOrganizationClick: () => void
     </>
   );
 };
+
+const ForceOrganizationSelectionFlow = () => {
```

Comment:
Super WIP, I'll push more commits to extract a couple of pieces here to reusable JSX chunks into the same module.